### PR TITLE
use Python3_EXECUTABLE instead of unspecified "python3" when creating aimet wheels

### PR DESCRIPTION
**Pull request by FelixSchwarz**
Using an unspecified "python3" might not work when rebuilding the aimet wheel. Also it is inconsistent with the rest of the build process.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -309,7 +309,7 @@ if(NOT remote_url STREQUAL "")
     # Run the install target first
     COMMAND "${CMAKE_COMMAND}" --build . --target install
     # Now run the packaging target to generate wheel files
-    COMMAND ${CMAKE_COMMAND} -DAIMET_PACKAGE_PATH=${AIMET_PACKAGE_PATH} -DSOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR} -DENABLE_CUDA=${ENABLE_CUDA} -DENABLE_TENSORFLOW=${ENABLE_TENSORFLOW} -DENABLE_TORCH=${ENABLE_TORCH} -DENABLE_ONNX=${ENABLE_ONNX} -DPATCHELF_EXE=${PATCHELF_EXE} -P ${CMAKE_CURRENT_SOURCE_DIR}/packaging/package_aimet.cmake
+    COMMAND ${CMAKE_COMMAND} -DPython3_EXECUTABLE=${Python3_EXECUTABLE} -DAIMET_PACKAGE_PATH=${AIMET_PACKAGE_PATH} -DSOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR} -DENABLE_CUDA=${ENABLE_CUDA} -DENABLE_TENSORFLOW=${ENABLE_TENSORFLOW} -DENABLE_TORCH=${ENABLE_TORCH} -DENABLE_ONNX=${ENABLE_ONNX} -DPATCHELF_EXE=${PATCHELF_EXE} -P ${CMAKE_CURRENT_SOURCE_DIR}/packaging/package_aimet.cmake
```
Comment (quic-bharathr) [inline CMakeLists.txt]: Can we use upper-case for the cmake flag per our convention?
`-DPYTHON3_EXECUTABLE=${Python3_EXECUTABLE}`

File: packaging/package_aimet.cmake [hunk 2]
```diff
@@ -192,7 +192,11 @@ foreach(package ${package_name_list})
       COMMAND find ${build_packaging_dir} -name "libaimet_tf_ops*.so" -exec ${PATCHELF_EXE} --set-rpath $ORIGIN:$ORIGIN/../../tensorflow:$ORIGIN/../../tensorflow/python {} \;
   )
   # Invoke the setup tools script to create the wheel packages.
-  execute_process(COMMAND python3 setup.py sdist bdist_wheel ${CUDA_OPTION} WORKING_DIRECTORY ${build_packaging_dir} OUTPUT_VARIABLE output_var)
+  execute_process(
+    COMMAND ${Python3_EXECUTABLE} setup.py sdist bdist_wheel ${CUDA_OPTION}
```
Comment (quic-bharathr) [inline packaging/package_aimet.cmake]: Change to "PYTHON3_EXECUTABLE" if you make the above requested change

**Review by quic-bharathr — CHANGES_REQUESTED** [review]
Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (FelixSchwarz) [reply CMakeLists.txt]: sure

**Review by quic-bharathr — APPROVED** [review]

File: packaging/package_aimet.cmake (hunk 2 above)
Comment (FelixSchwarz) [reply packaging/package_aimet.cmake]: yes, of course.

**Review by quic-akhobare — APPROVED** [review]

**Comment by quic-akhobare** [comment]
Looks good. I approved it. Thanks @FelixSchwarz

**Comment by FelixSchwarz** [comment]
Thank you both. Now I just need someone to merge it for me.

**Pull request merged**